src/gui_common/fonts/Thrive-Regular-Humongous.tres

```diff
@@ -13,8 +13,9 @@
 [ext_resource path="res://assets/fonts/NotoSansBengali-SemiBold.ttf" type="DynamicFontData" id=11]
 
 [resource]
-size = 65
+size = 32
```

Comment:
Is this font used anywhere else? I'd really prefer a new font size to be made. I wouldn't call 32 size even humongous anymore...